### PR TITLE
Use router-view in portfolio section

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: vue-frontend/src/pages/portfolio/[id].vue [hunk 1]
```diff
@@ -87,8 +98,12 @@ export default {
         this.meta.og.url = this.response.seoData.url;
       }
     },
+    scrollToPortfolio() {
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/portfolio/[id].vue:101]: Let's rename it to `scrollToTop`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/PortfolioSection.vue [hunk 2]
```diff
@@ -30,7 +30,24 @@
...
+              class="v2-normal-2-text v2-button tw-flex tw-items-center tw-w-fit"
+            >
+              <span class="tw-mr-2.5">VIEW</span>
+              <font-awesome-icon
+                class="tw-text-black-900 fa"
+                icon="arrow-right"
+                id="leftArrow"
+            /></router-link>
+
+            <a
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/PortfolioSection.vue:45]: I think here also we can use `router-link` instead of `a`

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/PortfolioSection.vue (hunk 2 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/home-new/PortfolioSection.vue:45]: Currently out smileplus portfolio is not ready so that's why i use anchor tag

Comment (cp-sumi-k) [reply vue-frontend/src/components/home-new/PortfolioSection.vue:45]: I understand that, but is it possible to use `router-link` for redirecting to smileplus page?

Comment (cp-shruti-s) [reply vue-frontend/src/components/home-new/PortfolioSection.vue:45]: Router path is not getting the smileplus website url

**Review by cp-sumi-k — APPROVED** [review]

File: vue-frontend/src/pages/portfolio/[id].vue [hunk 3]
```diff
@@ -87,8 +98,15 @@ export default {
         this.meta.og.url = this.response.seoData.url;
       }
     },
-  },
+    scrollToTop() {
+      this.response = null;
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/portfolio/[id].vue:102]: I think watchers are better then making response null

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**